### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by CadillacJack42**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1,8 +1,64 @@
...
+// Outputs
+const wins = document.getElementById('correct-guess');
+const losses = document.getElementById('incorrect-guess');
+const totals = document.getElementById('total-guess');
 
+// initialize global state
+let win = 0;
+let total = 0;
+const cups = [cup1, cup2, cup3];
+const buttons = [btn1, btn2, btn3];
```
Comment (BenFrancka) [inline app.js:21]: Nice approach by holding onto your elements in arrays!

Comment (BenFrancka) [inline app.js:21]: However, it appears you are not utilizing this buttons array.  Consider getting rid of unused variables.

File: app.js [hunk 2]
```diff
@@ -1,8 +1,64 @@
...
-  // update DOM to reflect the new state
+btn1.addEventListener('click', () => {
+    handleClick(cups[0], cups);
+});
+btn2.addEventListener('click', () => {
+    handleClick(cups[1], cups);
+});
+btn3.addEventListener('click', () => {
+    handleClick(cups[2], cups);
+});
```
Comment (BenFrancka) [inline app.js:31]: These are some nice, clean event listeners.  Great job extrapolating your logic into functions.

File: app.js (hunk 1 above)
Comment (CadillacJack42) [reply app.js:21]: I thought I pushed that. Must have forgot. Thanks for reminding me. I merged that change.

**Pull request merged**